__tests__/__snapshots__/App-test.js.snap

```diff
@@ -445,7 +445,7 @@ exports[`Button Example Page 1`] = `
                           "textAlign": "center",
                         },
                         {
-                          "color": "rgb(0, 122, 255)",
+                          "color": "silver",
```

Comment:
If the test is passing, does this mean that "SystemColorButtonFaceColor" is a valid windowsbrush?

Reply:
Yup - I think it is resulting in the same color as 'silver'